enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/BoltInteraction.java

```diff
@@ -71,6 +71,7 @@
     public BoltInteraction( Neo4jWithSocket server ) throws IOException
     {
         this.server = server;
+        server.restartDatabase( settings -> {} );
```

Comment:
This ended up like that because of my adding two procedures for testing, which are compiled to a temporary jar, meaning that I needed the TemporaryFolder Rule. However this means that the Neo4jWithSocket cannot be started before the temporary folder has been made. The way to control that order was to not have Neo4jWithSocket as a rule but to explicitly start it using the restartDatabase command. Anyway, the new commit should improve on that. Really think I should add a comment on this?